### PR TITLE
add sbt 2.x cross build setting

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

File: plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala [hunk 1]
```diff
@@ -4,7 +4,9 @@ import java.io.OutputStreamWriter
 import java.nio.file.Path
 
 import sbt.Keys._
-import sbt._
+// format: off
```
Comment (kitbellew) [inline plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala:7]: why is this necessary? if this is scala3 syntax, better to modify scalafmt configuration, no?

**Review by kitbellew — COMMENTED** [review]
@xuwei-k what is this change for?

@tgodzik can you please review? this is not something I'm familiar with.

File: plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala (hunk 1 above)
Comment (xuwei-k) [reply plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala:7]: https://github.com/scalameta/sbt-scalafmt/blob/0344d19d9d6e9c8d8944d9408f7774c79046c332/.scalafmt.conf#L23-L24

without this comment expand import

```diff
- import sbt.{given, _}
+ import sbt._
+ import sbt.given
```

but error if Scala 2.12

- https://github.com/scala/scala/pull/9736
- https://github.com/sbt/sbt/issues/7737

Comment (kitbellew) [reply plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala:7]: is this is an unreported bug in scalafmt?

**Review by tgodzik — APPROVED** [review]
Look good as far as I can see. Thanks @xuwei-k !
